tests/sles4sap/publiccloud/qesap_terraform.pm

```diff
@@ -119,6 +119,17 @@ sub run {
         # but not implemented in file_content_replace() yet poo#120690
         $escaped_token =~ s/\&/\\\&/g;
         set_var("HANA_TOKEN", $escaped_token);
+
+        if (get_var('PTF_INSTALL')) {
+            my $ptf_files = qesap_az_list_container_files(
+                storage => get_required_var('HANA_ACCOUNT'),
+                container => (split("/", get_required_var('HANA_CONTAINER')))[0],
+                token => get_required_var('HANA_TOKEN'),
+                prefix => 'ptf'
+            );
+            set_var("PTF_FILES", $ptf_files);
+            set_var("QESAP_AZ_CONTAINER", (split("/", get_required_var('HANA_CONTAINER')))[0]);
```

Comment:
The same as the before comment. 
Just in case we absolutely need to use `set_var`, please rename the setting not to start with `QESAP_` as this prefix hase a precise meaning in this repo:
- none of the `QESAP_` is dinamically populated with `set_var`
- `QESAP_` are only used as input for code in `lib/qesapdeployment.pm`

Reply:
Same answer as before

Reply:
Renaming done